CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changes
 
+## edge-21.5.2
+
+This edge release updates the proxy-init container to check whether the iptables
+rules have already been added, which prevents errors if the proxy-init container
+is restarted. Also, the linkerd cli now has tab completion for kubernetes
+resources, saving you precious keystrokes! Finally, the proxy has been updated
+with several fixes and improvements.
+
+* Added instructions to `build.md` for using a locally built proxy
+  (thanks @jroper!)
+* Added support for kubernetes resource aware tab completion
+* Updated `proxy-init` to skip configuring firewall if rules exists
+* Fixed viz uninstall to delete all RBAC objects (thanks @aryan9600!)
+* Improved diagnostics for rejected profile discovery
+* Added proxy logic to set the l5d-client-id header
+* Reduced proxy resource usage when there are no profiles
+* Updated proxy to fail connections when HTTP detection fails
+* Changed proxy to assume meshed connections are HTTP/2
```

Comment:
Not sure where this note comes from either.. we've used protocol hints in the proxy to assume meshed connections are HTTP/2 so this shouldn't be new.

Reply:
This note comes from [linkerd2-proxy/pull/982](https://github.com/linkerd/linkerd2-proxy/pull/982)

Reply:
Okay so again this is specific to the admin server. I think this should just be combined with the other mentioned change about the admin server.

Reply:
👍

Reply:
@kleimkuhler since these two items are specific to the admin server, are they user-facing enough to be included? I'd be fine with removing them if they will have no impact on users

Reply:
I think they are probably still worth keeping as a bullet even though it is small changes. I think they can be consolidate into:

```
* Changed the admin server to assume all meshed connections are HTTP/2 and fail connections when that is not the case
```